### PR TITLE
emit ...param:any[]

### PR DESCRIPTION
**Pull request by liudongpu**

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -234,7 +234,7 @@ interface EventEmitter extends EventEmitterListener {
      *
      *   emitter.emit('someEvent', 'abc'); // logs 'abc'
      */
-    emit(eventType: string): void
+    emit(eventType: string,...param:any[]): void
```
Comment (alloy) [inline types/react-native/index.d.ts]: Fix the linter failure like so: `emit(eventType: string, ...param: any[]): void`

**Review by alloy — CHANGES_REQUESTED** [review]
Thanks! Change looks good to me 👍 Just need to fix the linter failure.

**Comment by mhegazy** [comment]
@liudongpu can you please address the CI test failures

File: types/react-native/index.d.ts (hunk 1 above)
Comment (gyzerok) [inline types/react-native/index.d.ts]: And should maybe variable name be `params`?

Comment (alloy) [inline types/react-native/index.d.ts]: Good point

**Pull request merged**